core/lib/instrument.c

```diff
@@ -7596,9 +7596,17 @@ dr_prepopulate_cache(app_pc *tags, size_t tags_count)
 }
```

Comment:
Please add an entry to the changelist of new features at https://github.com/DynamoRIO/dynamorio/blob/master/api/docs/release.dox#L226

Reply:
done